### DISCUSSION TITLE
Streaming data, in general!

### DISCUSSION THREAD

**Discussion by christineberger** [MaterializeInc/mz-hack-day-2022#12] [Show and tell]
First off, I want to thank the Materialize team for hosting this event - I did **not** have extensive knowledge on tools which help in streaming transformations, but have followed the work that Emily did at Drizly in my attempts to understand the underlying functionality and use cases more! 

I wanted to join the hack day both to support the Materialize and Redpanda teams and because I wanted to know more about how Materialize and Redpanda solve various puzzles in the data space! Since I'm a consultant, I feel this is important information to wrap my head around.

I was surprised at how easy the sample was to spin up, and really appreciated this as I was on limited time for participating. Because of that, I was able to make it through the entire demo that Marta showed.

Although I didn't get to explore it extensively like all the other really great projects here, I definitely have a much better understanding about where Redpanda and Materialize sit in the stack and their usage, and I think now that I have the sample project, I'll be able to explore this with more confidence when I get the chance!

**Comment by JLDLaughlin** [comment]
Thank you so much for joining, @christineberger!

**Comment by morsapaes** [comment]
> I was on limited time for participating

We're so glad you still did, @christineberger! Thank you (also for sharing your experience)!
